packages/app/src/cli/services/app/config/link.ts

```diff
@@ -1,31 +1,29 @@
-import {saveCurrentConfig} from './use.js'
+import {setCurrentConfigPreference} from './use.js'
```

Comment:
This is changed a lot. It's doing the same thing but in a hopefully saner order. I'd read the new version and ignore the diff